perun-base/src/main/java/cz/metacentrum/perun/core/api/Auditable.java

```diff
@@ -219,34 +206,31 @@ public int compareByModifiedAt(Auditable auditable) {
 		try {
 			date1 = BeansUtils.getDateFormatter().parse(modifiedAt);
 		} catch (Exception ex) {
-			throw new InternalErrorRuntimeException("There is problem with parsing createdAt in object " + this,ex);
+			throw new InternalErrorRuntimeException("There is problem with parsing createdAt in object " + this, ex);
 		}
 
 		Date date2;
 		try {
 			date2 = BeansUtils.getDateFormatter().parse(auditable.getModifiedAt());
 		} catch (Exception ex) {
-			throw new InternalErrorRuntimeException("There is problem with parsing createdAt in object " + auditable,ex);
+			throw new InternalErrorRuntimeException("There is problem with parsing createdAt in object " + auditable, ex);
 		}
 
 		return date1.compareTo(date2);
 	}
+	*/
 
 	@Override
 	public String toString() {
-		StringBuilder str = new StringBuilder();
-
-		return str.append(this.getClass().getSimpleName()).append(":[").append(
-				"id='").append(this.getId()).append('\'').append(
-				']').toString();
+		return this.getClass().getSimpleName() + ":[" +
+				"id='" + this.getId() + '\'' +
+				']';
```

Comment:
No, it is not. Concatenation of constant number of strings is by the compiler silently converted to using StringBuilder, so in fact both ways are identical in function, but mine is shorter. The change was suggested by IntelliJ IDEA.